src/LSST/control/Ilc.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ *  This file is part of LSST M2 support system package.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "control/Ilc.h"
+
+// System headers
+
+// Project headers
+#include "system/Config.h"
+#include "util/Bug.h"
+#include "util/Log.h"
+
+using namespace std;
+
+namespace LSST {
+namespace m2cellcpp {
+namespace control {
+
+AllIlcs::AllIlcs(bool useMocks) {
+    // FUTURE: Once C API is available, create instances capable of communicating
+    //         with the FPGA.
+    if (!useMocks) {
+        throw util::Bug(ERR_LOC, "Only Mock instances available.");
+    }
+
+    // The first 6 ILC's are Tangential
+    // 7-78 are Axial.
```

Comment:
I have a question for this. Why are the first 6 ILCs tangential ? I know the actuators 73 - 78 are the tangential actuators. Do you mean they are 1-6 in ILC instead?
@pcortes80 Can you help to double check the FPGA code?

Reply:
This is a good question. I think I got that from a document, but I'm having tourble finding it now.

Reply:
Found it. 
[LSST-ILC-M2 Mirror Cell](https://docs.google.com/document/d/1Smg_C1Cdsj619dTNZM5Ec44D_DzFn-gD/edit)
Page 8, section 2.4.2 describes the broadcast message '66' to set the actuators and says 1-6 are tangential.

That contradicts 
[LTS-346 ICS Communications Protocol for M2 Support System ICD](https://docs.google.com/document/d/1e7stLeKYsmtxPNHrBjFEeprvzMUsqFJp/edit)
Page 11-12 in this document say broadcast '66' 73 thru 78 are tangential.

The second one is newer, and hopefully correct. I'll change the code to set 73-78 as the tangential and @pcortes80 can hopefully verify that is correct.

Reply:
According to what I found in my previous research, I think tangential actuators must be from 73-78. I added the Table 1 in the [Calculating the Total Net Forces](https://confluence.lsstcorp.org/display/LTS/Calculating+the+Total+Net+Forces) document. I will double-check this with the simulator.